packages/shared/src/hooks/usePaddlePayment.ts

```diff
@@ -109,6 +109,13 @@ export const usePaddlePayment = ({
               }),
             });
 
+            if (router.pathname.includes('/helloworld')) {
+              localStorage.setItem(
+                'funnelSubscribed',
```

Comment:
Let's store this string in a const to make it easier to be consistent across files 🙏